README.md

```diff
@@ -111,6 +111,21 @@ Pet Rescue is one of many projects initiated and run by Ruby for Good. You can f
 ## Pet Rescue Adoption Sites
 [Baja Pet Rescue](https://www.bajapetrescue.com)
 
+# 🌟 Core Values
+This is essentially where culture can start which is what we promote, demote, or flat out say no to. These should be the guiding principle when we interact with others, stakeholders, and future collaborators. A analogy that can be used is to say vision is the destination, stragety is how you'll get there, and core values is what you use to handle when things change or there is uncertainity (and they often do!).
+
+Here are our core values defined by early contributors and leads:
+
+### Code Quality and Collaboration
+Write maintainable code that is accessible to beginners, fostering a supportive environment for contributions while ensuring long-term sustainability to keep the project alive.
+
+### Communication and Perspective:
+Prioritize clear communication, embracing diverse viewpoints, and engaging stakeholders to gather feedback, with a commitment to timely responses and ongoing improvement.
+
+### Engagement and Practicality:
+Build partnerships with stakeholders, seeking their active involvement, while focusing on the practical impact of our work and resolving disagreements through constructive dialogue.
+
+
```

Comment:
These are great! 

They do a good job at covering a broad range of things. If we need more clarity in the future we can alawys revisist to be more specific. But, I like this approach and think it will work.

Reply:
More suggested edits:

Core values (defined by early contributors and leads):

**Code Quality and Collaboration:**
Write maintainable code that is accessible and enjoyable (to beginners and seasoned coders alike), supports and encourages contributors and their contributions, and ensures long-term sustainability of this project and the efforts it supports.

**Communication and Perspective:**
Prioritize clear communication, embrace diverse viewpoints, and always engage feedback, all with a commitment to timely responses and ongoing improvement for all. Rescue and adoption partner perspectives will be prioritized over abstracted concepts regarding their needs.

**Engagement and Practicality:**
Build upon stakeholder partnerships to foster and encourage their active involvement, focusing constructive discussion and dispute resolution on the practical impact of our work.

Reply:
Love these thanks @maebeale! You beat ChatGPT any day jeje